.nvmrc

```diff
@@ -0,0 +1 @@
+v14.18.2
```

Comment:
Question: would it make sense to pin to a version name or major version number than to this specific one?

No strong opinion just thinking out loud.

Reply:
That's also an option, I'd stick with a major pinned version like `v14.*` or `v16.x` since is the new LTS, thougths?

Reply:
Let's go with `v14` then